### DISCUSSION TITLE
Unable to create a student azure account

### DISCUSSION THREAD

**Discussion by AmanJulka** [github/india#712] [Students]
Hello FRT Team,
  I have my GitHub student pack active. I have completed my project for the FRT internship.

  I am unable to create a free student  Azure account to host my project.
The deadline to submit the project is on 20th January 2022. 

  How can host my project? Please help. 
Thank you. I hope you will look into this matter ASAP.
My Github project repo:- https://github.com/AmanJulka/WeatherHub_FRT_Project.git 
Yours sincerely,
Aman Julka

**Comment by triad7** [comment]
Hello FRT Team, I'm facing the same issue, everytime I create a resource
it's taking me to a page where it shows you can't avail the free student
dev azure account. Please help.
